a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -189,15 +189,16 @@ def execute(self, loadbalancer):
             return None
         if vthunder.undercloud:
             use_parent_part = CONF.a10_global.use_parent_partition
-            if use_parent_part and not vthunder.hierarchical_multitenancy:
-                LOG.warning("Hierarchical multitenancy is disabled, use_parent_partition "
-                            "configuration will not be applied for loadbalancer: %s",
-                            loadbalancer.id)
-            elif use_parent_part and vthunder.hierarchical_multitenancy:
-                parent_project_id = utils.get_parent_project(
-                    vthunder.project_id)
-                if parent_project_id:
-                    vthunder.partition_name = parent_project_id[:14]
+            if use_parent_part:
+                if vthunder.hierarchical_multitenancy == 'enable':
+                    parent_project_id = utils.get_parent_project(
+                        vthunder.project_id)
+                    if parent_project_id:
```

Comment:
We should raise a warning here or maybe even error out. The user has configured it to have the object created in the parent partition, but if none is found then we just create in the normal one. And then that normal partition can't be deleted without logging into the box....

Reply:
Ok Will error it out here, for `None` `parent_project_id`

Reply:
Added `LOG.warning` and resources are created in existing project id partition itself. 
![image](https://user-images.githubusercontent.com/52992745/92729401-e53bdd00-f38f-11ea-89a0-2be05c1c01de.png)

Reply:
**_Update_**:

Raised an Error here, instead of warning.

![image](https://user-images.githubusercontent.com/52992745/92874748-f20b1000-f425-11ea-8b52-a9e3234ddb31.png)